app/src/main/java/com/poklad/androidifystore/data/remote/StoreApi.kt

```diff
@@ -0,0 +1,30 @@
+package com.poklad.androidifystore.data.remote
+
+import com.poklad.androidifystore.data.remote.model.ProductItemResponse
+import com.poklad.androidifystore.utils.ApiConstants.ALL_PRODUCTS
+import com.poklad.androidifystore.utils.ApiConstants.CATEGORIES
+import com.poklad.androidifystore.utils.ApiConstants.CATEGORY
+import com.poklad.androidifystore.utils.ApiConstants.CATEGORY_NAME
+import com.poklad.androidifystore.utils.ApiConstants.PRODUCT_ID
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface StoreApi {
+    @GET(ALL_PRODUCTS)
+    suspend fun getAllProducts(): List<ProductItemResponse>
+
+    @GET("$ALL_PRODUCTS/{$PRODUCT_ID}")
```

Comment:
Create a constant for this constructed string in `ApiConstants` something like `ApiConstants.GET_PRODUCT_BY_ID `or similar

Reply:
done